recipes-extended/jailhouse/jailhouse-imx_git.bb

```diff
@@ -38,6 +38,8 @@ CELL_DIR ?= "${JH_DATADIR}/cells"
 CELLCONF_DIR ?= "${JH_DATADIR}/configs"
 INMATES_DIR ?= "${JH_DATADIR}/inmates"
 
+TUNE_CCARGS:remove:mx93-nxp-bsp = "-mcpu=cortex-a55"
```

Comment:
> Do you think it would be a good idea to reset `TUNE_CCARGS`? Additionally, do you believe this issue might occur with other processors?

This is not the best solution. It's better to find a way to disable using FPU for all arm64 CPU on the jailhouse side.

For now, the march `armv8-a+nofp` is used for any arm64 target (https://github.com/nxp-imx/imx-jailhouse/commit/14c8802f1d2b784e7cda14fc4a2cd649f2296fdb). The only incompatible with this `march` `mcpu` target is `cortex-a55` which seems present only in imx93.
So, yeah, its more like a workaround to pass building.